airbyte-integrations/connectors/source-google-ad-manager/sample_files/configured_catalog.json

```
@@ -0,0 +1,80 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "ad_unit_per_hour_report_stream",
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-04/schema#",
+          "type": "object",
+          "properties": {
+            "cpm_cpc_revenue": {
+              "title": "Cpm Cpc Revenue",
+              "type": "integer"
+            },
+            "impressions": { "title": "Impressions", "type": "integer" },
+            "eCpm": { "title": "Ecpm", "type": "integer" },
+            "unfilled_impressions": {
+              "title": "Unfilled Impressions",
+              "type": "integer"
+            },
+            "ad_unit": { "title": "Ad Unit", "type": "string" },
+            "hour": { "title": "Hour", "type": "integer" },
+            "date": { "title": "Date", "type": "string", "format": "date-time" }
+          },
+          "required": [
+            "cpm_cpc_revenue",
+            "impressions",
+            "eCpm",
+            "unfilled_impressions",
+            "ad_unit",
+            "hour",
+            "date"
+          ]
+        },
```

```suggestion
        "json_schema": {},
```
the variable `json_schema` can be simplify here.

---

Done, but I don't know why we are doing this.
Do you mind explaining?

---

For integration tests the connector won't check the schema using the configured catalog. It will validate based on schema files.